qutebrowser/utils/usertypes.py

```diff
@@ -246,6 +247,52 @@ def reset(self):
                      'err_config', 'err_key_config'], is_int=True, start=0)
 
 
+# Where a download should be saved
+class DownloadTarget:
```

Comment:
I think what seems most weird to me (and probably other people used to read Python) is how a class is abused as a namespace and the whole thing is called an enum :wink:

What about just having entirely separate `FileDownloadTarget`, `FileObjDownloadTarget`, and `OpenDownloadTarget` classes? Sure, it's a bit weird how they're only connected to each other via their name, but it still seems like a more obvious solution to me.